### ISSUE TITLE
Remove deprecated t.spread.pb.noOperation and printTraceback

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5370]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5370|
|Type|task|
|Created|2011-11-05 20:35:01Z|
|Branch|https://github.com/twisted/twisted/tree/pb-deprecations-5370|

`noOperation` and `printTraceback` in [twisted/spread/pb.py](twisted/spread/pb.py) were deprecated in Twisted 8.2 ([#3054](https://github.com/twisted/twisted/issues/3054)) and they can be removed.

<details><summary>Searchable metadata</summary>

```
trac-id__5370 5370
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_pb_deprecations_5370 branches/pb-deprecations-5370
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1320525301000000 1320525301000000
changetime__1321294325000000 1321294325000000
version__None None
owner__thijs thijs
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

(In [33082]) Branching to 'pb-deprecations-5370'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [33083]) Remove tests, add news file. refs [#5370](https://github.com/twisted/twisted/issues/5370)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

Removed related code/tests and ran pyflakes.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Thanks for working on this, thijs.

Code and [builds](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/pb-deprecations-5370) look good, please merge!

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [33149]) Merge pb-deprecations-5370: remove printTraceback and noOperation from twisted.spread.pb, deprecated since Twisted 8.2.

Author: thijs
Reviewer: jesstess
Fixes: [#5370](https://github.com/twisted/twisted/issues/5370)
